radixdlt/src/main/java/com/radixdlt/consensus/bft/BFTEventReducer.java

```diff
@@ -99,6 +100,47 @@
 		void sendTimeoutProcessed(View view, BFTNode leader);
 	}
 
+	static class RTTStatistics {
```

Comment:
Add TODO comment to move this to outside of Reducer at some point

Reply:
I'll just move it now.